flang/test/Lower/OpenACC/acc-private.f90

```diff
@@ -1,10 +1,13 @@
 ! This test checks lowering of OpenACC loop directive.
 
-! RUN: bbc -fopenacc -emit-fir %s -o - | FileCheck %s
+! RUN: bbc -fopenacc -emit-fir %s -o - | FileCheck %s --check-prefixes=CHECK,FIR
+! RUN: bbc -fopenacc -emit-hlfir %s -o - | FileCheck %s --check-prefixes=CHECK,HLFIR
 
 ! CHECK-LABEL: acc.firstprivate.recipe @firstprivatization_ref_50xf32 : !fir.ref<!fir.array<50xf32>> init {
 ! CHECK: ^bb0(%{{.*}}: !fir.ref<!fir.array<50xf32>>):
 ! CHECK:   %[[ALLOCA:.*]] = fir.alloca !fir.array<50xf32>
+! FIXME: we need hlfir.declare here to satisfy the assumptions about
+!        the HLFIR lowering, i.e. that every varible has fir/hlfir.declare.
```

Comment:
Thank you for the review, Valentin!  I wonder if you agree with this note.  I do not think it is a blocker for HLFIR enabling, but I would like to create an issue to track it, if it makes sense.

Reply:
Yes I think your note is correct. I can update the recipe generation for that.

Reply:
Thanks!